provider/javascript.go

```diff
@@ -88,46 +88,37 @@ func (p *Javascript) BoolGetter() func() (bool, error) {
 	}
 }
 
-func (p *Javascript) setParam(param string, val interface{}) error {
+func (p *Javascript) paramAndEval(param string, val any) error {
 	err := p.vm.Set(param, val)
 	if err == nil {
 		err = p.vm.Set("param", param)
 	}
 	if err == nil {
 		err = p.vm.Set("val", val)
 	}
+	if err == nil {
```

Comment:
Warum brauchts das? Use case?

Reply:
Brauchen, brauchts das nicht. Es ist nur im Javascript Plugin so, und ich wollte die beiden Implementierungen möglichst gleich haben, da ich wieder an den transforms am basteln bin und die natürlich in beiden plugins möglichst gleich einbauen will.

Reply:
Habe das aus dem PR Rausgenommen